Automattic/wp-calypso
# o2 blocks: Add task and project status blocks

#### Changes proposed in this Pull Request

* Adds a task block based on previous work by @mtias 

#### Testing instructions

* Check this branch.
*  Run `npx lerna run build --scope="@automattic/o2-blocks"`
*  Upload the resulting `editor.js` `editor.sccs`,  and `editor.rtl.sccs`  from the `apps/o2-blocks/dist` dir to your sandbox under `xxx-plugins/xxx-blocks/dist`.
* Sandbox a P2.
* Create a post.
* Include the Task block
  <img src="https://user-images.githubusercontent.com/746152/70479980-278a7200-1b11-11ea-84f6-3b7f566f05ea.png" height=150/>

   
* Expect it to  look like this:
    <img src="https://user-images.githubusercontent.com/746152/70479766-a632df80-1b10-11ea-9cb1-ca8c2c7ab264.png" height=200/>

 ### Tasks

- [ ] Add start / end date.
- [ ] Color for task.
- [ ] Generate anchor.
